docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-en.md

```diff
@@ -0,0 +1,105 @@
+# Deploy Dashboard
+
+This topic will introduce how to install and deploy Dashboard through tar package in detail.
```

Comment:
package here is a singular noun. please add a before it.
It's better to say a .tar package or a TAR package since it is a file extension.